tests/e2e/ligo-plugin-e2e-tests.spec.ts

```diff
@@ -55,6 +55,32 @@ describe('Ligo Plugin E2E Testing for Taqueria CLI', () => {
 		await cleanup();
 	});
 
+	test('ligo plugin will support cli options', async () => {
+		const { execute, cleanup, writeFile, readFile, ls, spawn } = await prepareEnvironment();
+		const { waitForText } = await spawn('taq', 'init test-project --debug');
+		await waitForText("Project taq'ified!");
+
+		const { stdout } = await execute('taq', 'install ../taqueria-plugin-ligo', './test-project');
+		expect(stdout).toContain('Plugin installed successfully');
+
+		const { stdout: stdout2 } = await execute('taq', '--version', './test-project');
+		expect(stdout2).not.toEqual([]);
+
+		const { stdout: stdout3 } = await execute('taq', '--build', './test-project');
+		expect(stdout3).not.toEqual([]);
```

Comment:
Should we also assert that nothing was output to stderr?

Reply:
added